validation-test/BuildSystem/dsymutil_jobs.test

```diff
@@ -0,0 +1,10 @@
+# RUN: %empty-directory(%t)
+# RUN: mkdir -p %t
+# RUN: SKIP_XCODE_VERSION_CHECK=1 SWIFT_BUILD_ROOT=%t %swift_src_root/utils/build-script --dry-run --darwin-install-extract-symbols --dsymutil-jobs 5 --cmake %cmake 2>&1 | %FileCheck %s
+
+# REQUIRES: standalone_build
```

Comment:
I think that using `llvm-dsymutil` would be preferable unless you want to restrict the test to macOS.

Reply:
Thanks -- given that code only run in macOS, restricted the test in 0d0199fe249